include/swift/AST/KnownProtocols.h

```diff
@@ -41,6 +42,12 @@ enum : uint8_t {
 enum : unsigned { NumKnownProtocolKindBits =
   countBitsUsed(static_cast<unsigned>(NumKnownProtocols - 1)) };
 
+using KnownProtocolSet = FixedBitSet<NumKnownProtocols, KnownProtocolKind>;
+
+/// Produces a set of all protocols that have an inverse, i.e., for every
+/// known protocol KP in the set, ~KP exists.
+KnownProtocolSet getInvertableProtocols();
```

Comment:
Typo: invertible not invertable. But I think this isn't a great name. Perhaps it should also be a global constant instead of a function?

Reply:
What about calling these "removable protocols"?

Reply:
I'm going with the pitch's terminology (which I also invented myself) for now to get this prototyped. Previous iterations of this used the term "Suppressible". Given that there's also going to be the notion of subtraction (i.e., `Type ~ Copyable`) to remove defaults coming from `Type`, I think using type inverse/subtraction terminology is at least aligned.